lib/core.ps1

```diff
@@ -1,22 +1,14 @@
-# Note: The default directory changed from ~/AppData/Local/scoop to ~/scoop
-#       on 1 Nov, 2016 to work around long paths used by NodeJS.
-#       Old installations should continue to work using the old path.
-#       There is currently no automatic migration path to deal
-#       with updating old installations to the new path.
-$scoopdir = $env:SCOOP, "$env:USERPROFILE\scoop" | Select-Object -first 1
+. "$PSScriptRoot/config.ps1"
```

Comment:
This will break the current installer

Reply:
Good catch, didn't considering that the old installer just downloading a single `core.ps1`. It's hard to move on now..